go.mod

```diff
@@ -101,3 +101,5 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	howett.net/plist v1.0.1 // indirect
 )
+
+replace github.com/elastic/elastic-agent-libs => github.com/ycombinator/elastic-agent-libs v0.0.0-20250926122109-345a0542c79e
```

Comment:
TODO: remove once https://github.com/elastic/elastic-agent-libs/pull/359 is merged